README.md

```diff
@@ -146,7 +146,7 @@ $tax = new Tax(
 );
 ```
 
-This is shiny declarative code, but the biggest advantage is laziness. No one calculation will happen until `amount` method will be called.
+This is a shiny declarative code, but the biggest advantage is laziness. No one calculation will happen until the `amount` method will be called.
```

Comment:
"a shiny" is wrong

Reply:
![image](https://user-images.githubusercontent.com/606705/103202789-68b01880-4904-11eb-9571-2e7514b5de0a.png)

Reply:
![image](https://user-images.githubusercontent.com/606705/103202809-782f6180-4904-11eb-866c-a43d9ca8ead0.png)